utilities/openemr-cmd/openemr-cmd

```diff
@@ -11,6 +11,7 @@ DOCKER_EXEC_CMD='docker exec -i'                                 # docker exec c
 CONTAINER_ID=$(docker ps | grep _openemr | cut -f 1 -d " ")      # get the container id
```

Comment:
check out my comment below on what to place in CONTAINER_ID

Reply:
sorry, what is that mean of this one?

Reply:
I guess I am thinking something like here:
```
INSANE_DEV_DOCKER="_openemr-7-3-312_1"
EASY_DEV_DOCKER="_openemr_1"
CONTAINER_ID=$(docker ps | grep $INSANE_DEV_DOCKER | cut -f 1 -d " ")
if [ -z $CONTAINER_ID ]
then
  CONTAINER_ID=$(docker ps | grep $EASY_DEV_DOCKER | cut -f 1 -d " ")
fi
```
Then the script would work for both easy dev and insane dev.

Reply:
yep, but seems that not only "_openemr-7-3-312_1" version.  
```
# chkdocker-names openemr
=======Running Docker Names=======
docker_openemr-7-1_1       <<
docker_openemr-7-1-redis_1
docker_openemr-7-2_1          <<
docker_openemr-7-2-redis_1
docker_openemr-7-3-311_1   <<
docker_openemr-7-3-312_1   <<
docker_openemr-7-3-redis-311_1
docker_openemr-7-3-redis-312_1
docker_openemr-edge_1
```

Reply:
I mean if only need to check _openemr-7-3-312_1 that is ok, if it needs to switch to check multiple versions, seems not work

Reply:
In the insane development environment, there is one "brain" docker that does all the work (it does the building), which generally uses the most recent alpine version (the easy dev docker also uses this alpine version), which is why there only needs to be one docker to default to if it exists for the insane dev environment. About every 6 months when alpine releases a new version, then would update it on the script.

To support forcing use of other dockers (for example when looking at docker log), then would consider looking into the getops utility posted by @jesdynf, so then command may look something like this if wish to force a docker name (of course, for default behavior would not need it):
```
openemr-cmd -d docker_openemr-7-3-redis-312_1 dl
```
Usage: openemr-cmd [-d docker-name] [ command and command parameters ]

Reply:
Hi @bradymiller, thanks for your explanation. So I will add the default to it.
```
INSANE_DEV_DOCKER="_openemr-7-3-312_1"
EASY_DEV_DOCKER="_openemr_1"
CONTAINER_ID=$(docker ps | grep $INSANE_DEV_DOCKER | cut -f 1 -d " ")
if [ -z $CONTAINER_ID ]
then
  CONTAINER_ID=$(docker ps | grep $EASY_DEV_DOCKER | cut -f 1 -d " ")
fi
```

And also add -d option for support the other dockers. btw, -d idea is really cool. I working on it.